core/templates/services/math-interactions.service.ts

```diff
@@ -61,9 +61,27 @@ export class MathInteractionsService {
         'not valid.';
     }
     if (errorMessage.indexOf('is not a valid postfix operator.') !== -1) {
+      let humanReadableOperator = (
+        errorMessage[0] === '*' ? '×' :
+        errorMessage[0] === '/' ? '÷' :
+        errorMessage[0]);
       errorMessage = (
-        'Your answer seems to be missing a variable/number after the "' +
-        errorMessage[0] + '".');
+        'Your answer seems to be missing a variable/number after the ' +
+        humanReadableOperator + ' operator.');
```

Comment:
This part seems repeated, should we consider using a function?

Reply:
@kevintab95 Unless we consider using a function `humanReadableOperator` How can we update `* as ×` and `/ as ÷`.

Reply:
Hm, they don't seem to be repeated now, so it's OK to ignore this comment.